airbyte-integrations/bases/base-normalization/build.gradle

```diff
@@ -38,6 +36,10 @@ tasks.named('check').configure {
     dependsOn generate
 }
 
+tasks.named("jar").configure {
+    dependsOn copySshScript
+}
```

Comment:
This is odd. Let me look into this and get back to you.

Reply:
What's really missing is
```
tasks.named('assemble').configure {
    dependsOn generate
}
```
In any case it doesn't matter, this package is going to removed from the gradle build with https://github.com/airbytehq/airbyte/pull/35306
I'm told these images are dead.